front/admin/db.ts

```diff
@@ -54,6 +62,11 @@ async function main() {
   await AssistantAgentMessage.sync({ alter: true });
   await AssistantMessage.sync({ alter: true });
 
+  await AssistantAgentConfiguration.sync({ alter: true });
+  await AssistantAgentGenerationConfiguration.sync({ alter: true });
+  await AssistantAgentRetrievalConfiguration.sync({ alter: true });
+  await AssistantAgentDatasourceConfiguration.sync({ alter: true });
```

Comment:
I wonder if this should be renamed from `AssistantAgentDatasourceConfiguration` to `AssistantAgentRetrievalDatasourceConfiguration` cause the datasource configuration are used for retrieval, but do we really want var names that are longer than the entire Lord of the Rings trilogy?

Reply:
I guess it's not necessary because if there is a datasource then it has to be some retrieval-related thing ? I even wonder if the `Agent` is needed at all in the name